Create task_1

anflx opened the pull request.

anflx committed:
Create task_1

dmitrii-kruglov commented on task_1:
```diff
@@ -0,0 +1,36 @@
+  cin >> first.chisl; 
+  cout << "Введите знаменатель дроби ";
+  cin >> first.znam;
+  int delitel = NOD(first.chisl, first.znam);
```
задание не выполнено. Нужно было написать **функцию**, которая принимает в качестве аргумента структуру. Осталось немного, просто вынесите 30-35 в отдельную функцию, вместо вывода перезапишите в поля структуры. 

А вывод будет иметь вид:
  cout << "Сокращенная дробь:" << endl;
  cout << first.chisl << endl;
  cout << "__" << endl;
  cout << first.znam << endl;

anflx committed:
Update task_1

Пыталась сделать как вы и сказали + то, что скинули в телегу, оно все равно не работает, эта void функция не понимает откуда брать значения и что такое переменная delitel. Пробовала выносить функцию NOD в мейн, вместо cout в 36 и 38 строчках писать New(first.chisl) и New(drob) каждый раз с & и без этого символа, так тоже не работает

anflx committed:
Create task_2

dmitrii-kruglov commented on task_1:
```diff
@@ -0,0 +1,39 @@
+
+void New(struct & drob){
+  int delitel = NOD(first.chisl, first.znam);
+  first.chisl = first.chisl / delitel;
```
ошибка компиляции

dmitrii-kruglov commented on task_2:
```diff
@@ -0,0 +1,35 @@
+    cin >> array[i]; 
+  }
+
+  void NewArray(int *array){
```
почему функция внутри другой функции? В каком компиляторе это делаете?

dmitrii-kruglov reviewed (changes requested):
Обе задачи не компилируются
